### PR TITLE
Added icon to web interface

### LINKED ISSUE #57: Add a favicon
The demo site is currently missing a favicon.

A custom icon could be designed or a fitting free icon could be used. Twemoji is 100% free for use, so an emoji can be taken from there.

### PR DESCRIPTION
**Pull request by MagicLike**
## Description

I added an Twemoji as an icon for the web interface & updated the README with the licence attribution.

Fixes #57 

### Type of change

- [ ] Bug fix (added a non-breaking change which fixes an issue)
- [ ] New feature (added a non-breaking change which adds functionality)
- [x] Updated documentation (updated the readme, templates, or other repo files)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

- [ ] Ran tests with `composer test`
- [ ] Added or updated test cases to test new features
<br>

- [x] Tested on own Heroku instance

## Checklist:

- [x] I have checked to make sure no other pull requests are open for this issue
- [x] The code is properly formatted and is consistent with the existing code style
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

new icon

### CODE REVIEW

**Comment by MagicLike** [comment]
I know it's pretty simple, but I think that it could be used until a proper logo or something else is made.

**Review by DenverCoder1 — APPROVED** [review]
This looks great!

Possibly we could change the name of the file from`2328.png` to something like `favicon.png`?

File: src/demo/index.php [hunk 1]
```diff
@@ -24,6 +24,7 @@ function gtag() { dataLayer.push(arguments); }
     <script type="text/javascript" src="./js/toggle-dark.js" defer></script>
     <script type="text/javascript" src="./js/jscolor.min.js" defer></script>
     <script async defer src="https://buttons.github.io/buttons.js"></script>
+    <link rel="icon" href="2328.png">
```
Comment (DenverCoder1) [inline src/demo/index.php:27]: The type should be specified for better browser compatibility
```suggestion
    <link rel="icon" type="image/png" href="2328.png">
```

**Comment by MagicLike** [comment]
Done. 👍

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for the contribution :tada:

**Pull request merged**